Lab6.ipynb

```diff
@@ -0,0 +1,443 @@
+      "text/plain": [
+       "<Figure size 1296x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "mixed = np.zeros(r.size)\n",
+    "for i in range(mixed.size):\n",
+    "    if i < signal.size:\n",
+    "        mixed[i] = np.add(r[i], signal[i])\n",
+    "    else:\n",
+    "        mixed[i] = r[i]\n",
+    "plt.hist(mixed,bins = 450)\n",
+    "plt.title('Data After Signal Injection')\n",
+    "plt.xlabel('Measurement')\n",
+    "plt.ylabel('Counts')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The mean has moved from 250 to 350. The distance it has moved to the right, 100, is exactly the mean of the background distribution, and this is no coincidence. On average, the background produces 100 particle events, so if on average the signal, independent of the background, produces 250 particle events, then of course since both events are happening at the same time, the overall number of observed particles in an average signal event, when we take the background into account, will be the sum of the particles from the background plus the particles from the signal, which on average is going to be 250 + 100 = 350. So it makes perfect sense that the mean moved 100 units to the right for the signal. It retained its shape as a Poisson distribution because the sum of Poisson distributions is a Poisson distribution, naturally. The net effect of all of this is that the observed measurement is not a measurement of the signal, but a measurement of the signal plus background, so that  the observed value is more likely to be stronger than the true injected signal. Even though it's a poisson distribution, due to the central limit theorem, the signal looks gaussian. So for the large sample size I have, it seems symmetric. "
```

Comment:
Problem 2B:
- Math makes sense.
- Graph axes labels and overall graph looks good. No typos there and shapes/mean shift for the signal make sense.
- Second sentence is a bit long, try breaking it into two for readability? The reasoning totally makes sense and is good for explanation of question.
- Good pointing out the shape is retained. Good pointing out symmetry. Did you mention anywhere that it is not biased?
- I didn't catch any typos, nothing really to change here! My only suggestion could maybe be discussing how the background can add a little more, or a little less, due to the shape of the background, which is why the signal also has a spread of possible values? (Or discuss this later too.) Otherwise seems great.